setup.py

```diff
@@ -40,13 +40,12 @@
 INSTALL_REQUIRES = ['numpy >= 1.10.1',
                     'pandas >= 0.16.0',
```

Comment:
min version change

Reply:
Sorry that got reverted in some of my testing.